person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def _age?
```

Comment:
- [ ] Great job on implementing the Person class! It's evident that you have a good understanding of object-oriented programming in Ruby. However, there are some discrepancies between your implementation and the project requirements. 
- Please kindly make the age? method private as required in the project requirements. To make the _age? method private, you can add the private keyword above its definition like so:

```ruby
class Person
  attr_accessor :name, :age
  attr_reader :id

  def initialize(age, name = 'Unknown', parent_permission = true)
    @id = Random.rand(1..1000)
    @name = name
    @age = age
    @parent_permission = parent_permission
  end

  def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end
end
```